### PR TITLE
Added fast-url-parser

### PR DESCRIPTION
**Pull request by sid226**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: f/fast-url-parser/fast-url-parser_ubi_8.6.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: fast-url-parser
+# Version	: 1.1.3
+# Source repo	: https://github.com/apache/fast-url-parser
+# Tested on	: UBI 8.6
+# Language      : JavaScript
```
Comment (seth-priya) [inline f/fast-url-parser/fast-url-parser_ubi_8.6.sh:8]: we have been typically using "node" for nodejs packages , let's do the same for consistency

File: f/fast-url-parser/fast-url-parser_ubi_8.6.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+PACKAGE_NAME=fast-url-parser
+PACKAGE_VERSION=${1:-v1.1.3}
+PACKAGE_URL=https://github.com/petkaantonov/urlparser.git
+CURDIR="$(pwd)"
+dnf install -y wget git yum-utils nodejs nodejs-devel nodejs-packaging npm 
+
+
+# Install fast-url-parser
+npm install $PACKAGE_NAME@$PACKAGE_VERSION
```
Comment (seth-priya) [inline f/fast-url-parser/fast-url-parser_ubi_8.6.sh:29]: why do we need this when we are installing from source later?

File: f/fast-url-parser/fast-url-parser_ubi_8.6.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+# run tests
+git clone $PACKAGE_URL
+cd urlparser
+git checkout $PACKAGE_VERSION
+npm install && npm audit fix && npm audit fix --force;
```
Comment (seth-priya) [inline f/fast-url-parser/fast-url-parser_ubi_8.6.sh:36]: this and "npm test" will need error handling , look at the template or some of the recently reviewed PRs for node packages, like https://github.com/ppc64le/build-scripts/pull/2764

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: f/fast-url-parser/fast-url-parser_ubi_8.6.sh (hunk 3 above)
Comment (sid226) [reply f/fast-url-parser/fast-url-parser_ubi_8.6.sh]: Added changes.

File: f/fast-url-parser/fast-url-parser_ubi_8.6.sh (hunk 2 above)
Comment (sid226) [reply f/fast-url-parser/fast-url-parser_ubi_8.6.sh]: Added changes.

File: f/fast-url-parser/fast-url-parser_ubi_8.6.sh (hunk 1 above)
Comment (sid226) [reply f/fast-url-parser/fast-url-parser_ubi_8.6.sh]: Added changes.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**